wp-content/themes/pub/wporg-learn-2024/src/card-course/view.js

```diff
@@ -0,0 +1,26 @@
+function init() {
+	/**
+	 * Enhance the accessibility of the card course template.
+	 *
+	 * This function adds an `aria-hidden="true"` and `tabindex="-1"`
+	 * to the post featured image block in the card course template.
+	 */
+	function enhanceCardCourseAccessibility() {
+		document
+			.querySelectorAll( '.wporg-learn-card-grid figure.wp-block-post-featured-image' )
+			.forEach( ( featuredImg ) => {
+				featuredImg.setAttribute( 'aria-hidden', 'true' );
+				featuredImg.setAttribute( 'tabindex', '-1' );
```

Comment:
I think we should only be doing this to the anchor element

Reply:
We could consider setting role="presentation" on the image, as the alt text is the same as the title and therefore it won't be adding any value.

Reply:
>  think we should only be doing this to the anchor element

Ah, yeah, thanks! 18a01a2

> We could consider setting role="presentation" on the image, as the alt text is the same as the title and therefore it won't be adding any value.

| Before | After |
|-|-|
| ![Screenshot 2024-07-26 at 08 38 59](https://github.com/user-attachments/assets/a2da98e4-1a05-4409-9844-53e36bda435a) | ![Screenshot 2024-07-26 at 08 46 03](https://github.com/user-attachments/assets/9c866bfc-8351-4877-b3ea-942336d91458) |